get_ecg.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+import math
+
+
+class GetEcg:
+
+    """
+    Creating this class to convert raw ECG data of all types into HR data
+    and clinical indication of brady-/tachycardia.
+    """
+    min2sec = 60
+
+    def __init__(self, csv_file=None, update_time=5,
+                 brady_threshold=60, tachy_threshold=100):
+        """
+        Constructor for ECG processing needs the input csv file to work on and
+        time for often to update the instantaneous HR
+
+        :param csv_file: the input file containing ECG data
+        :param update_time: instantaneous HR update time
+        :param brady_threshold: threshold for bradycardia
+        :param tachy_threshold: threshold for tachycardia
+        """
+        if csv_file is not None:
```

Comment:
You can refactor this as `if csv_file:`; that will evaluate as false if it is `None`.